src/visibility-and-privacy.md

```diff
@@ -177,7 +178,8 @@ pub mod outer_mod {
             inner_mod_visible_fn();
         }
 
-        // This function is visible
+        // This function is visible only within `inner_mod`,
+        // which is the same as leaving it private
```

Comment:
```suggestion
        // which is the same as leaving it private.
```

Reply:
Most of the other comments don't have a period. What's the convention here?

Reply:
The other comments should be fixed if so; we should strive for proper English sentences everywhere.